tests/cpython-tests/Makefile

```diff
@@ -23,20 +24,29 @@ ext2fs: appdir
 	$(MYST) mkext2 appdir ext2fs
 
 clean:
-	rm -fr appdir ext2fs @test_101*
+	rm -fr appdir ext2fs @test_101* hostfs
 
 tests:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test -f tests.passed -q
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test -f tests.passed.1 -q
+	$(RUNTEST) ./exec.sh
 
-run-single:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m test $(TC) -v
+#########################
+##	Dev targets
+#########################
+hostfs:
+	cp -r appdir/ hostfs/
 
-run-str:
-	$(MYST_EXEC) $(OPTS) ext2fs /cpython/python -c "$(PROG_STR)"
+run-list: $(FS)
+	$(MYST_EXEC) $(OPTS) $(FS) /cpython/python -m test -f $(TESTFILE) --timeout 30 -q
 
-pdb:
-	$(RUNTEST) $(MYST_EXEC) $(OPTS) ext2fs /cpython/python -m pdb /cpython/Lib/test/test_grammar.py -v 
+
+run-single: $(FS)
+	$(RUNTEST) $(MYST_EXEC) $(OPTS) $(FS) /cpython/python -m test $(TC) -q
+
+run-str: $(FS)
+	$(MYST_EXEC) $(OPTS) $(FS) /cpython/python -c "$(PROG_STR)"
+
+pdb: $(FS)
+	$(RUNTEST) $(MYST_EXEC) $(OPTS) $(FS) /cpython/python -m pdb /cpython/Lib/test/test_grammar.py -v
 
 ct:
-	rm -rf /tmp/myst*
\ No newline at end of file
+	rm -rf /tmp/myst* @test_101*
```

Comment:
`ct` is not used anymore?

Reply:
This is a dev target for cleaning up leaked resources.